src/Makefile.in

```diff
@@ -26,7 +26,9 @@ core_flags   += -Wwrite-strings
 core_cflags  += ${CFLAGS} -Wmissing-prototypes
 core_cxxflags = ${CXXFLAGS} -DHAVE_INTTYPES_H
 
+ifneq (${PLATFORM},emscripten)
 core_flags   += -I${PREFIX}/include
+endif
```

Comment:
I had a couple of thoughts on this. First, config.sh could just output something like `PREFIX:=` or `PREFIX:=.` into platform.inc that would not be likely to accidentally include anything undesirable. This is a little cheesy but involves the fewest changes.

The other thought I had was kind of a compromise between that solution and what you have here: set `PREFIX:=` or `PREFIX:=.` in platform.inc and then change this check to make sure `${PREFIX}` is set and/or isn't `.` instead of an arch-specific check.